spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject { Comment.new(text: 'This is a comment', author_id: 1, post_id: 1) }
+
+  before { subject.save }
+
+  it 'new comment should be saved in the database' do
+    expect(subject).to be_persisted
+  end
+
+  it 'text should be present' do
+    subject.text = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ]  You should fix the error raised while running comment model tests, and consider adding the custom  method for comment model `update_post_comments_counter` so that you app model can be fully tested properly.

Reply:
Hi @ishimwezachee, thank you for the feedback, I will be implementing all the custom methods for the models, however, when I run the test are passing